### PR TITLE
update solrconfig.xml for Sufia 6.2

### PR DESCRIPTION
**Pull request by weiweishi**
The solrconfig.xml file was updated in upstream Sufia 6.2 but was not included in our local 6.2 PR. This is to update the file.

### CODE REVIEW

**Comment by pgwillia** [comment]
Is this still needed?

**Comment by weiweishi** [comment]
Yes, and I have one update to add today.

*Weiwei Shi*

Digital Initiative Applications Librarian
University of Alberta Libraries
2-10L Cameron Library
Edmonton, Alberta, Canada T6G 2J8
Phone:(780)492-7802
Fax: (780)248-1209
Email: weiwei.shi@ualberta.ca

**Comment by weiweishi** [comment]
Changes to solrconfig.xml are completed. This branch is now ready to merge. 

Weiwei

File: solr_conf/conf/solrconfig.xml [hunk 1]
```diff
@@ -158,16 +169,27 @@
...
   </requestDispatcher>
 
   <requestHandler name="/analysis/field" startup="lazy" class="solr.FieldAnalysisRequestHandler" />
   <requestHandler name="/update" class="solr.UpdateRequestHandler"  />
+
   <updateHandler class="solr.DirectUpdateHandler2">
     <updateLog>
       <str name="dir">${solr.data.dir:}</str>
     </updateLog>
+    <autoCommit>
```
Comment (pgwillia) [inline solr_conf/conf/solrconfig.xml]: Just curious how the autocommit values were chosen?

Comment (weiweishi) [inline solr_conf/conf/solrconfig.xml]: It was copied from Plano's configuration file and are possibly set by
jetty.

*Weiwei Shi*

Digital Initiative Applications Librarian
University of Alberta Libraries
2-10L Cameron Library
Edmonton, Alberta, Canada T6G 2J8
Phone:(780)492-7802
Fax: (780)248-1209
Email: weiwei.shi@ualberta.ca

Comment (henryzhang87) [inline solr_conf/conf/solrconfig.xml]: Auto commit affects two things:
1) durability which is determined by hard commit
2) visibility which is determined by soft commit

For HydraNorth, my understanding is that it is a not a heavy indexing
application. So whenever a file is indexed, you might want it to be visible
to the users right away. So,
we might need to set OpenSearcher as true for soft commit (the side effects
are caches might be invalidated).

Please let me know what you care most and we can set up auto commit in a
way that suits your need.

Comment (pgwillia) [inline solr_conf/conf/solrconfig.xml]: It looks like these particular values were chosen to work well with
ActiveFedora
projecthydra/active_fedora:c9733e3015710cbbc6c73fa4a85b5c59f35fd773.  Based
on this I'm comfortable with merging this change.

**Pull request merged**